core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -26,28 +26,35 @@ import { ExplorationOpportunity } from '../opportunities-list-item/opportunities
 import constants from 'assets/constants';
 import { Subscription } from 'rxjs';
 
+type LoadOpportunities = Promise<{
```

Comment:
I'm not sure types should start with verbs, since they are not variables or functions. Can you think of a word for this which describes the type correctly and which is a noun?

Reply:
Done